skipchain/skipchain_test.go

```diff
@@ -609,12 +609,44 @@ func TestService_Propagation(t *testing.T) {
 	}
 	service := genService.(*Service)
 
-	sbRoot, err := makeGenesisRosterArgs(service, ro, nil, VerificationNone,
+	roster := onet.NewRoster(ro.List[:5])
```

Comment:
it would be easier to understand this test if these were renamed as: ro -> fullRoster, roster -> firstFiveRoster